source/install/desktop-app-changelog.rst

```diff
@@ -6,12 +6,80 @@ Desktop application changelog
 
 Latest Mattermost Desktop App releases:
 
-- `Release v5.2 <#id1>`_
-- `Release v5.1 <#id5>`_
-- `Release v5.0 <#id19>`_
-- `Release v4.7 <#id34>`_
-- `Release v4.6 <#id51>`_
-- `Release v4.5 <#id63>`_
+- `Release v5.3 <#id1>`_
+- `Release v5.2 <#id3>`_
+- `Release v5.1 <#id17>`_
+- `Release v5.0 <#id31>`_
+- `Release v4.7 <#id46>`_
+- `Release v4.6 <#id63>`_
+
+Release v5.3
+--------------
+
+Release Date: 
+
+**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/latest>`_
+
+Compatibility
+~~~~~~~~~~~~~~~
+
+- Desktop App is supported on any supported Extended Support Release or a newer Mattermost server version.
+- Support for Windows v8 and v8.1 have been dropped. Minimum supported Windows version was updated to 10+.
+- Updated Chromium minimum supported version to 110+.
+
+Highlights
+~~~~~~~~~~~~~~~
+
+- Added application diagnostics.
+- Implemented a global calls widget window.
+
+Improvements
+~~~~~~~~~~~~~~~
+
+All Platforms
+^^^^^^^^^^^^^
+
+- Added support for Gnome's "do-not-disturb" status.
+- Added a menu item for showing the logs folder.
+- Improved performance by reducing the number of calls for URL detection.
+- Changed the tray behavior on left click. Left-clicking on the system tray symbol now hides the application to system tray if it is already visible.
+- Defaulted to opening a file when it is clicked from the download list.
+
+Architectural Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Major version upgrade of Electron to v23.1.2. Electron is the underlying technology used to build the Desktop App.
+
+Bug Fixes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All Platforms
+^^^^^^^^^^^^^
+
+- Fixed an issue where a user could open a blank Electron window using the main window.
+- Fixed an issue where image thumbnails did not always display in the downloads for MAS builds.
+- Fixed an issue where the Boards/Playbooks tabs sometimes didn't appear automatically when a server was added.
+- Fixed an issue where RPM conflicted with other Electron based applications.
+- Fixed an issue where a websocket certificate wasn't trusted along with a HTTPS certificate for the same origin.
+- Fixed an issue where opening the app with a deeplink could cause the app not to redirect to the correct URL.
+- Fixed an issue with closing the downloads dropdown menu when clicking "Show in folder".
+- Fixed an issue with maximizing the main window when a monitor removed.
+- Fixed an issue where special characters in the server name caused the top bar to disappear.
+- Fixed an issue where OneLogin users wouldn't have their credentials remembered.
+- Fixed an issue with plugin navigation displaying a white empty bar.
```

Comment:
a white empty bar where?

Reply:
Clarified the text. (Related PR https://github.com/mattermost/desktop/pull/2446)